release-notes/opensearch-sql.release-notes-2.5.0.0.md

```diff
@@ -0,0 +1,96 @@
+Compatible with OpenSearch and OpenSearch Dashboards Version 2.5.0
+
+### Features
+
+#### Structure changes
```

Comment:
Please pay attention to the category naming: https://github.com/opensearch-project/opensearch-plugins/blob/main/RELEASE_NOTES.md#categories. Core team needs to map each category here to a consolidated release notes finally.

Reply:
ok thanks...